src/main/java/MockClass.java

```diff
@@ -5,4 +5,9 @@ public int doSomething(int seed) {
 
         return seed + 5;
     }
+
+    public static void main(String[] args) throws Exception {
+
+        throw new Exception("An issue.");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS00112)